app/uk/gov/hmrc/apiplatformjobs/repository/UnusedApplicationsRepository.scala

```diff
@@ -58,12 +58,12 @@ class UnusedApplicationsRepository @Inject()(mongo: ReactiveMongoComponent)(impl
       background = true)
   )
 
-  def applicationsByEnvironment(environment: Environment): Future[List[UnusedApplication]] = find("environment" -> environment)
+  def unusedApplications()(implicit environment: Environment): Future[List[UnusedApplication]] = find("environment" -> environment)
 
-  def applicationsToBeDeleted(environment: Environment, deletionDate: DateTime = DateTime.now): Future[List[UnusedApplication]] =
+  def unusedApplicationsToBeDeleted(deletionDate: DateTime = DateTime.now)(implicit environment: Environment): Future[List[UnusedApplication]] =
     find("environment" -> environment, "scheduledDeletionDate" -> Json.obj("$lte" -> deletionDate))
 
-  def deleteApplication(environment: Environment, applicationId: UUID): Future[Boolean] =
+  def deleteUnusedApplicationRecord(applicationId: UUID)(implicit environment: Environment): Future[Boolean] =
```

Comment:
does environment need to be implicit here? if its used in the query would it make more sense to have it as an argument to the function?